src/implementation/Client/HTTPClient/state.ts

```diff
@@ -103,6 +103,10 @@ export default class HTTPClientState implements IClientState {
       },
     });
 
+    if (result === "") {
+      return { results: [] };
```

Comment:
This fixes HTTP, please add gRPC, along with tests. You can also return it as `StateQueryResponseType`.
```suggestion
      return { results: [] } as StateQueryResponseType;
```